external/js/cothority/src/byzcoin/arrange/utils.ts

```diff
@@ -0,0 +1,32 @@
+
+/**
+ * Utility function to arrange arguments on contracts
+ */
+export class Utils {
+
+    // transform a protobuf encoded number to a number
```

Comment:
I don't think it's a protobuf, but rather a `VarInt`. Also, I would directly import the library from here:

https://github.com/chrisdickinson/varint/blob/master/decode.js

Which keeps the range check that is removed in this copy/paste...